lib/psd_formatter.rb

```diff
@@ -7,8 +7,7 @@ class PsdFormatter < Syslog::Logger::Formatter
   LINE_FORMAT = "(thread-%s) [%s] %5s -- : %s\n"
 
   def initialize(deployment_info)
```

Comment:
Deployed::DETAILS above gets passed in to the PsdFormatter initialiser in `logging.rb` which is setup in the deployment project.
- The OpenStruct here was redundant
- We now expect a hash, so switched to `values_at` instead
- Name was always nil, so removed it here

Reply:
Theres a bit of scope for improvement overall here. Lots of duplication of this kind of stuff, which probably belongs in a standard psd_tools gem. Also, burying the link between Deployed::DETAILS and PsdFormatter in the deployment project meant I almost missed it, and at first assumed DETAILS was unused. It was only on not finding where PsdLogger was setup that I realised the connection.

This isn't very 12factor app